pkg/plugin/server/manager.go

```
@@ -127,6 +132,27 @@ func (m *Manager) NewProxy(content *NewProxyContent) (*NewProxyContent, error) {
 	return content, nil
 }
 
+func (m *Manager) CloseProxy(content *CloseProxyContent) {
```

`error` should be returned here.

---

You can return the aggregate errors.

---

Errors are now returned, but not handled at call site, since that is at most some duplicated logs.